### PR TITLE
[Stack] Removing the local nugets and accommodating  a rename change in the SDK

### PR DESCRIPTION
**Pull request by bganapa**
## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by bganapa** [comment]
ps sign here - https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign/

**Comment by deathly809** [comment]
@bganapa Is this your signing job?  https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign/555/

File: src/StackAdmin/Azs.Subscriptions/Module/Azs.Subscriptions/Generated.PowerShell.Commands/SwaggerPathCommands/Get-AzsDelegatedProviderOffer.ps1 [hunk 1]
```diff
@@ -72,21 +72,21 @@ function Get-AzsDelegatedProviderOffer
     $ErrorActionPreference = 'Stop'
 
     $NewServiceClient_params = @{
-        FullClientTypeName = 'Microsoft.AzureStack.Management.Subscriptions.SubscriptionsManagementClient'
```
Comment (deathly809) [inline src/StackAdmin/Azs.Subscriptions/Module/Azs.Subscriptions/Generated.PowerShell.Commands/SwaggerPathCommands/Get-AzsDelegatedProviderOffer.ps1]: Why did you remove Management from the name of the client?

Comment (bganapa) [reply src/StackAdmin/Azs.Subscriptions/Module/Azs.Subscriptions/Generated.PowerShell.Commands/SwaggerPathCommands/Get-AzsDelegatedProviderOffer.ps1]: Alignment with Azure :)

Comment (deathly809) [reply src/StackAdmin/Azs.Subscriptions/Module/Azs.Subscriptions/Generated.PowerShell.Commands/SwaggerPathCommands/Get-AzsDelegatedProviderOffer.ps1]: What do you mean in alignment with Azure? 

https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Compute/Management.Compute/Generated/ComputeManagementClient.cs

**Comment by bganapa** [comment]
ps-sign - https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign/579/
poweshell-demand - https://azuresdkci.westus2.cloudapp.azure.com/job/powershell-demand/806/console

File: src/StackAdmin/Azs.Subscriptions/generate.ps1 [hunk 2]
```diff
@@ -1,6 +1,6 @@
 
-$rpName = "subscriptions"
-$name = "Subscriptions"
+$rpName = "subscription"
+$name = "Subscription"
 $location = Get-Location
 $psswagger = "D:\github\PSswagger"
```
Comment (markcowl) [inline src/StackAdmin/Azs.Subscriptions/generate.ps1]: We don't want this hard-coed path, right?

**Comment by markcowl** [comment]
@bganapa Do we need to make any changes to versions?

File: src/StackAdmin/Azs.Subscriptions/generate.ps1 (hunk 2 above)
Comment (bganapa) [reply src/StackAdmin/Azs.Subscriptions/generate.ps1]: yes, we don't want the hardcoded path, the script needs to change to clone or download the psswagger instead of taking its path.. I will file an issue and address it later

**Comment by bganapa** [comment]
@markcowl  Checks are successful. This should be good to merge
